test/test_mpi_tree.jl

```diff
@@ -8,6 +8,9 @@ include("test_trixi.jl")
 # pathof(Trixi) returns /path/to/Trixi.jl/src/Trixi.jl, dirname gives the parent directory
 const EXAMPLES_DIR = joinpath(pathof(Trixi) |> dirname |> dirname, "examples", "tree_2d_dgsem")
 
+# Needed to skip certain tests on Windows CI
+import ..CI_ON_WINDOWS
```

Comment:
```suggestion
CI_ON_WINDOWS = (get(ENV, "GITHUB_ACTIONS", false) == "true") && Sys.iswindows()
```
I know this is code duplication, but it makes life easier when running some tests locally by including specific test files.

Reply:
Good idea!